js/images.js

```diff
@@ -1,4 +1,4 @@
-import '../images/background.jpg'
-import '../images/chip.png'
-import '../images/visa.png'
-import '../images/mastercard.png'
\ No newline at end of file
+//import '../images/background.jpg'
```

Comment:
source image files should be loaded directly in Pug template or in SCSS, not in JS